rts/Lua/LuaTracyExtra.cpp

```diff
@@ -0,0 +1,82 @@
+/* This file is part of the Recoil engine (GPL v2 or later). */
+
+#include "LuaInclude.h"
+#include "LuaUtils.h"
+
+#include "LuaTracyExtra.h"
+#include "System/Misc/TracyDefs.h"
+
+#include <tracy/Tracy.hpp>
+#include <common/TracyQueue.hpp>
```

Comment:
Should have asked in the other PR, but I don't believe including this file is needed, I don't see anything from this header being actually used.

Reply:
Right, I just took whatever was in the old PR. There's also another unused `#include` directive for logging which I have used for debugging. I'll get rid of both.

Reply:
Looks like it's actually needed, here's what we use:
https://github.com/wolfpld/tracy/blob/dc4aa7a1262375dc808ec12572c6558f913ac80e/public/common/TracyQueue.hpp#L618-L622